### DISCUSSION TITLE
Algoritmo em C

### DISCUSSION THREAD

**Discussion by mario-baianor** [community/community#49442] [Codespaces] [answered]
### Select Topic Area

Question

### Body

Construa um algoritmo que peça as 4 notas bimestrais e mostre a média, com uma casa decimal usando linguagem C

**Comment by danisantosss** [comment, marked as answer]
#include <stdio.h>

int main() {
   float nota1, nota2, nota3, nota4, media;

   // Pede as notas bimestrais
   printf("Digite a primeira nota bimestral: ");
   scanf("%f", &nota1);

   printf("Digite a segunda nota bimestral: ");
   scanf("%f", &nota2);

   printf("Digite a terceira nota bimestral: ");
   scanf("%f", &nota3);

   printf("Digite a quarta nota bimestral: ");
   scanf("%f", &nota4);

   // Calcula a média
   media = (nota1 + nota2 + nota3 + nota4) / 4.0;

   // Mostra a média com uma casa decimal
   printf("A média bimestral é: %.1f\n", media);

   return 0;
}
